bokehjs/src/lib/models/tickers/log_ticker.ts

```diff
@@ -61,20 +61,24 @@ export class LogTicker extends AdaptiveTicker {
         }
       }
     } else {
-      const startlog = Math.ceil(log_low * 0.999999)
-      const endlog = Math.floor(log_high * 1.000001)
+      const sign_log_low = Math.sign(log_low)
+      const sign_log_high = Math.sign(log_high)
+      const d = 0.000001
+      const startlog = Math.ceil(log_low * (1.0-sign_log_low*d))
+      const endlog = Math.floor(log_high * (1.0+sign_log_high*d))
```

Comment:
Just a suggestion, but I would find something like this a little easier to follow, re: intention
```suggestion
      const d = 0.000001
      const low_pad = 1.0 - Math.sign(log_low)*d
      const high_pad = 1.0 + Math.sign(log_high)*d 
      const startlog = Math.ceil(log_low * low_pad)
      const endlog = Math.floor(log_high * high_pad)
```